cloud-controller-manager/do/certificates_test.go

```diff
@@ -0,0 +1,318 @@
+/*
+Copyright 2017 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package do
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/digitalocean/godo"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+type kvCertService struct {
+	store         map[string]*godo.Certificate
+	reflectorMode bool
```

Comment:
We do not seem to set `reflectorMode` anywhere, can we remove it?

When in doubt (and FWIW), I generally prefer populating fakes with the right values over setting a special mode (except for simulating error cases, but that's not what we have here).